test/Remote-Sim/args_parsing.cpp

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
```

Comment:
at some point we need to figure out how to separate `nvq++` compile / execution tests (these and NVQPP folder) from the MLIR / LLVM FileCheck regression tests  (comment / question from today's walk through).

Reply:
Indeed, I've posted https://github.com/NVIDIA/cuda-quantum/issues/1089 to make sure we'll do it at some point.